### ISSUE TITLE
kebechet.exception.InternalError: Failed to retrieve version information for dependency igitt, (dev: False)

### ISSUE THREAD

**Issue by fridex** [thoth-station/kebechet#78]
From logs:

```python
2018-07-24 11:17:09,973 [1] DEBUG    kebechet.config: Using token '***************************************'
--
  | 2018-07-24 11:17:09,973 [1] INFO     kebechet.config: Running manager 'update' for 'thoth-station/kebechet'
  | 2018-07-24 11:17:09,973 [1] INFO     kebechet.utils: Cloning repository git@github.com:thoth-station/kebechet.git to /tmp/tmpxzawajqy
  | 2018-07-24 11:17:10,459 [1] INFO     kebechet.managers.update: Using Pipfile for dependency management
  | 2018-07-24 11:17:12,492 [1] DEBUG    kebechet.source_management: Issue 'No dependency management found' not found, not closing it
  | 2018-07-24 11:17:15,011 [1] DEBUG    kebechet.source_management: Issue 'Failed to perform initial lock of software stack' not found, not closing it
  | 2018-07-24 11:17:15,017 [1] ERROR    kebechet.config: An error occurred during run of manager 'update' <class 'kebechet.managers.update.UpdateManager'> for thoth-station/kebechet, skipping
  | Traceback (most recent call last):
  | File "/usr/local/lib/python3.6/site-packages/kebechet/config.py", line 129, in run
  | kebechet_manager(slug, ServiceType.by_name(service_type), service_url, token).run(labels)
  | File "/usr/local/lib/python3.6/site-packages/kebechet/managers/update.py", line 595, in run
  | result = self._do_update(labels, pipenv_used=True)
  | File "/usr/local/lib/python3.6/site-packages/kebechet/managers/update.py", line 490, in _do_update
  | old_direct_dependencies_version = self._get_direct_dependencies_version()
  | File "/usr/local/lib/python3.6/site-packages/kebechet/managers/update.py", line 168, in _get_direct_dependencies_version
  | version = cls._get_dependency_version(dependency, is_dev=is_dev)
  | File "/usr/local/lib/python3.6/site-packages/kebechet/managers/update.py", line 99, in _get_dependency_version
  | f"Failed to retrieve version information for dependency {dependency}, (dev: {is_dev})")
  | kebechet.exception.InternalError: Failed to retrieve version information for dependency igitt, (dev: False)

```

**Comment by fridex** [comment]
Hint: case sensitive/insensitive naming (?)

**Comment by fridex** [comment]
> Hint: case sensitive/insensitive naming (?)

So it turned out it is #82, a workaround fix in #81
